src/main/java/apoc/text/Strings.java

```diff
@@ -573,4 +573,10 @@ public String toCypher(@Name("value") Object value, @Name(value = "config",defau
         }
         return null;
     }
+
+    @UserFunction
+    @Description("apoc.text.repeat(item, count) - string multiplication")
+    public String repeat(@Name("item") String item, @Name("count") long count) {
+        return new String(new char[(int) count]).replace("\0", item);
```

Comment:
better just use a pre-sized StringBuilder and a loop with append.
as the item might be longer than length = 1